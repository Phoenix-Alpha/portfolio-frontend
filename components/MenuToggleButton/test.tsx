import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { MenuToggleButton } from './index'

describe('<MenuToggleButton />', () => {
  it('should render correct html with default props', () => {
    const onButtonClick = jest.fn()

    const wrapper = shallow(<MenuToggleButton onButtonClick={onButtonClick} />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
