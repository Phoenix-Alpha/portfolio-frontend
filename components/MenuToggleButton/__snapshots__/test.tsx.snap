// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuToggleButton /> should render correct html with default props 1`] = `
<button
  aria-label="Toggle Menu"
  className="lg:hidden h-full focus:outline-none focus:shadow-outline"
  onClick={[MockFunction]}
>
  <ToggleMenuIcon />
</button>
`;
